app/services/shopify_import/data_parsers/customers/base_data.rb

```diff
@@ -0,0 +1,20 @@
+module ShopifyImport
+  module DataParsers
+    module Customers
+      class BaseData
+        def initialize(shopify_customer)
+          @shopify_customer = shopify_customer
+        end
+
+        def customer_attributes
```

Comment:
@vfonic I think we should name this `user_attributes` as we change (shopify) custometr to (spree) user.